### PR TITLE
Set the version like in the scikit-learn  (issue #104) 

### PR DESCRIPTION
**Pull request by igormusinov**

### CODE REVIEW

File: everware/authenticator.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 from traitlets import Unicode, Set
 from traitlets.config import LoggingConfigurable
-
+from .__init__ import __version__
```
Comment (anaderi) [inline everware/authenticator.py]: why not just `from . import __version__`?

File: everware/user_wait_handler.py [hunk 2]
```diff
@@ -50,7 +50,8 @@ def get(self, name, user_path):
                 html = self.render_template(
                     "spawn_pending.html",
                     user=current_user,
-                    need_wait=int(is_done)
+                    need_wait=int(is_done),
+                    __version__=__version__
```
Comment (anaderi) [inline everware/user_wait_handler.py]: name of the argument starting with `__` seems a bit odd. Why not just `current_version` or `package_version` or `version`?

File: everware/authenticator.py (hunk 1 above)
Comment (igormusinov) [reply everware/authenticator.py]: ok

**Review by anaderi — CHANGES_REQUESTED** [review]
please consider simplification as suggested by individual comment

File: everware/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.9.0"
+version = "0.9.0"
```
Comment (anaderi) [inline everware/__init__.py]: wait, here underscores for `__version__` is quite normal (it is the case for every package I've seen). I was complaining only about using underscores for function argument names. 
sorry for the confusion.

**Pull request merged**